vms/platformvm/block/executor/helpers_test.go

```diff
@@ -395,67 +394,81 @@ func defaultFx(clk *mockable.Clock, log logging.Logger, isBootstrapped bool) fx.
 	return res
 }
 
-func buildGenesisTest(ctx *snow.Context) []byte {
-	genesisUTXOs := make([]api.UTXO, len(preFundedKeys))
+func buildGenesisTest(ctx *snow.Context) *genesis.Genesis {
```

Comment:
I actually didn't reailze we have two genesis helpers to create pre-funded genesis. Wondering if it makes sense for us to consolidate this into a single place somewhere and export it out of a package.

Reply:
This is what I do in https://github.com/ava-labs/avalanchego/pull/2107 for genesis helpers and subsequent PRs for other test helpers.
Sorry we got the code duplication here, just trying to segment changes limit PRs scope

Reply:
Also historically, duplication arised when I repackaged platformvm but the way test parameters were set were slightly different across packaged. https://github.com/ava-labs/avalanchego/pull/2107 and subsequent PR remove duplicated code and fixes test to make them work with the default values and not ony with the specific values used in different packages.